core/common/src/MapLayerSettings.ts

```diff
@@ -6,6 +6,7 @@
  * @module DisplayStyles
  */
 
+import { compareStringsOrUndefined } from "@bentley/bentleyjs-core";

```

Comment:
Need to remove this import now (linter will complain).

Reply:
done.